feat: LEAP-1776: Add Pauses API

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/13157887204)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/399
<!-- Follow Merge -->
Hi @robot-ci-heartex!

This PR was automaticaly generated via Follow Merge. 
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.

- https://github.com/HumanSignal/label-studio-sdk/pull/399 linked by [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/13011317742)

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13011317742

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13012875342

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13013763852

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13017314444

triklozoid commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13058549724)
> Successfully merged:  96 files changed, 1044 insertions(+), 5047 deletions(-)

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1776'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13058549724

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13062946272

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13155987330

triklozoid committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13156033055

triklozoid commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13156114007)
> Successfully merged:  49 files changed, 814 insertions(+), 175 deletions(-)

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1776'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13156114007

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13157081222

robot-ci-heartex committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13157138858

nikitabelonogov commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13157887204)

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1776'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13157887204

triklozoid committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13157887204
